include/deal.II/base/symmetric_tensor.templates.h

```diff
@@ -308,8 +308,9 @@ namespace internal
                   AssertThrow(
                     false,
                     ExcMessage(
-                      "No convergence in iterative QL eigenvector algorithm.")) return std::
-                    array<std::pair<Number, Tensor<1, dim, Number>>, dim>();
+                      "No convergence in iterative QL eigenvector algorithm."));
+                  return std::array<std::pair<Number, Tensor<1, dim, Number>>,
+                                    dim>();
```

Comment:
Clearly nobody has looked at these lines in many years. That alone is a fascinating fact, I figure that every line of code is seen every once in a while at least...